### PR TITLE
Some small performance change 2

### PR DESCRIPTION
**Pull request by ggggamer**

### CODE REVIEW

**Comment by ggggamer** [comment]
Sorry for submit these change in two pull request.

File: cocos2dx/actions/CCActionEase.h [hunk 1]
```diff
@@ -407,7 +407,7 @@ class CC_DLL CCEaseElasticInOut : public CCEaseElastic
 class CC_DLL CCEaseBounce : public CCActionEase
 {
 public:
-    float bounceTime(float time);
+    static float bounceTime(float time);
```
Comment (zilongshanren) [inline cocos2dx/actions/CCActionEase.h:410]: why change a member function to static function? This change will break back compatiability.

File: cocos2dx/label_nodes/CCLabelAtlas.cpp [hunk 2]
```diff
@@ -131,6 +131,7 @@ void CCLabelAtlas::updateAtlasValues()
 
     CCAssert( n <= m_pTextureAtlas->getCapacity(), "updateAtlasValues: Invalid String length");
     ccV3F_C4B_T2F_Quad* quads = m_pTextureAtlas->getQuads();
+	ccColor4B c = { _displayedColor.r, _displayedColor.g, _displayedColor.b, _displayedOpacity };
```
Comment (zilongshanren) [inline cocos2dx/label_nodes/CCLabelAtlas.cpp]: The indentation is wrong here. please use 4 spaces as tab. thanks.

File: cocos2dx/label_nodes/CCLabelAtlas.cpp [hunk 3]
```diff
@@ -159,19 +160,21 @@ void CCLabelAtlas::updateAtlasValues()
         quads[i].br.texCoords.u = right;
         quads[i].br.texCoords.v = bottom;
 
-        quads[i].bl.vertices.x = (float) (i * m_uItemWidth);
+		float beginX = (float)(i * m_uItemWidth);
```
Comment (zilongshanren) [inline cocos2dx/label_nodes/CCLabelAtlas.cpp]: The indentation is wrong here. please use 4 spaces as tab. thanks.

File: cocos2dx/actions/CCActionEase.h (hunk 1 above)
Comment (ggggamer) [inline cocos2dx/actions/CCActionEase.h:410]: This function can be static , so changed to static.
Does this has back compatiability porblem?

Comment (zilongshanren) [inline cocos2dx/actions/CCActionEase.h:410]: Yes, object or object pointer can't access static member function via . or ->

Comment (ggggamer) [inline cocos2dx/actions/CCActionEase.h:410]: OK

**Pull request closed without merging**